packages/commons/src/events/FieldConfig.ts

```diff
@@ -223,10 +226,22 @@ const LocationOptions = z.object({
   type: z.enum(['ADMIN_STRUCTURE', 'HEALTH_FACILITY', 'CRVS_OFFICE'])
 })
 
+const AdministrativeArea = BaseField.extend({
+  type: z.literal(FieldType.ADMINISTRATIVE_AREA),
+  options: AdminAreaOptions
+}).describe('Administrative area input field')
+
 const Location = BaseField.extend({
-  type: z.literal(FieldType.LOCATION),
-  options: LocationOptions
-}).describe('Location input field')
+  type: z.literal(FieldType.LOCATION)
+})
+
+const Facility = BaseField.extend({
+  type: z.literal(FieldType.FACILITY)
+})
+
+const Office = BaseField.extend({
+  type: z.literal(FieldType.OFFICE)
+})
```

Comment:
Creating different kinds of fields will not let us search multiple types of locations e.g.- [facitlity, location]

Reply:
This `Facility` or `Office` type of field is an input field which shows a list of health facility kind of locations or offices in the dropdown. But suppose, if we had to search the office and facilities from a single input field, we can't do that from here.